### PR TITLE
Adding PR template

### PR DESCRIPTION
**Pull request by zashary**
# Description
Adding default PR template to add guidance to PR submission

# Solution
Add PR template doc to main branch will set this by default on all pull requests committed to the repo.

### CODE REVIEW

**Comment by tnoff** [comment]
This needs to be in the `.github` directory https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Summary
```
Comment (tnoff) [inline pull_request_template.md:1]: We should add some placeholder values of what we expect here

**Comment by zashary** [comment]
> This needs to be in the `.github` directory https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

Yes, you can put it in the root directory based on that guidance. But I went ahead and created the .github directory and put it there. I thought it could potentially increase confusion since there's a local .git dir already.

File: pull_request_template.md (hunk 1 above)
Comment (zashary) [reply pull_request_template.md]: Updated

**Review by tnoff — APPROVED** [review]

**Pull request merged**